javascript/apps/taiga/src/assets/editor/comment.css

```diff
@@ -0,0 +1,11 @@
+/*
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+*/
+
+body {
+  margin-block: 0.5rem;
```

Comment:
Use a spacing variable here (--spacing-8)

Reply:
I can't. This is a CSS file for the editor that is not compiled because it is outside the Angular context.